_maps/17-richard_barad.md

```diff
@@ -0,0 +1,9 @@
+---
+day: 17
+creator: Richard Barad
+title: BlueBikes Bike Share Trips in Boston Metro Area durring the week of July 10 - July 14th 2023
+thumbnail: /assets/thumbnails/17-richard-barad.png
+source: https://github.com/richardbarad/bikemap
+---
+
+The map looks at the most common bike trip routes using the Boston Blue Bikes network. The data shown is for the week of July 11th 2023 to July 15th, all routes with more than three trips are included. The thickness and color of the line indicate the number of trips along a given route. Routes with many frequent daily trips include routes from South Boston to Downtown and rides within Cambridge. 
```

Comment:
**suggestion:** Just adding an image of the map into the preview. By default, if you don't provide a description, then the thumbnail image gets shown. But if you do provide a description, then you have to also provide any images you want to show alongside it.

```suggestion
The map looks at the most common bike trip routes using the Boston Blue Bikes network. The data shown is for the week of July 11th 2023 to July 15th, all routes with more than three trips are included. The thickness and color of the line indicate the number of trips along a given route. Routes with many frequent daily trips include routes from South Boston to Downtown and rides within Cambridge. 

![the map described above](assets/thumbnails/17-richard-barad.png)
```